src/3d/qgstilingscheme.h

```diff
@@ -60,10 +60,18 @@ class _3D_EXPORT QgsTilingScheme
     //! Returns CRS of the tiling scheme
     QgsCoordinateReferenceSystem crs() const { return mCrs; }
 
+    /**
+     * Returns the full extent used in the constructor
```

Comment:
It may be worth adding that the "full extent" does not need to have square shape - and that the root tile will be a square that contains the "full extent"